tests/data/pyret.arr

```diff
@@ -0,0 +1,22 @@
+//! 22 lines 8 code 3 comments 6 blanks
```

Comment:
```suggestion
# 22 lines 8 code 3 comments 6 blanks
```
Would want a Pyret comment there.

Reply:
Thanks, I’m not sure how the copy and paste mangled that.